### ISSUE TITLE
RGB timeout changed!

### ISSUE THREAD

**Issue by dhruv8sh** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#47]
Model : Nitro AN515-56
 Script started working fine recently until I noticed that the RGB timeout was very little (about 3 secs).

**Comment by JafarAkhondali** [comment]
Wait ... what? what do you mean by timeout :)) 
I have no idea why it didn't work, and why it's working now

**Comment by dhruv8sh** [comment]
So I had [https://github.com/JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module/issues/43](this issue) open because it wasn't working on my model. Turns out, it works fine on EndeavourOS and not on Manjaro(both with default packages). Probably, something related to packages.

By a 3 sec timeout, I mean that when I am inactive/away from the keyboard, the backlight only stays on for 3 secs.

Thanks for this amazing project, btw!

**Comment by JafarAkhondali** [comment]
@dhruv8sh I think you can change this setting in bios.

**Comment by dhruv8sh** [comment]
Yes @JafarAkhondali , I am aware of the setting to turn off/on the backlight timeout, but the problem is I want it on but I don't want it too short...  Is there a variable I can edit to change the timeout?

I, however, will change this from windows for now.
